setup.py

```diff
@@ -36,25 +41,20 @@ def _path_under_setup(*args):
 for k, v in list(env.items()):
     env[k] = os.environ.get('%s_%s' % (pkg_name.upper(), k), v)
 
+is_win = True if 'win' in platform.platform().lower() else False
 
-USE_CYTHON = os.path.exists(_path_under_setup(pkg_name, '_levmar.pyx'))
-# Cythonize .pyx file if it exists (not in source distribution)
-ext_modules = []
-
-if len(sys.argv) > 1 and '--help' not in sys.argv[1:] and sys.argv[1] not in (
-        '--help-commands', 'egg_info', 'clean', '--version'):
-    import numpy as np
-    ext = '.pyx' if USE_CYTHON else '.c'
-    ext_modules = [Extension('%s._levmar' % pkg_name,
-                             [os.path.join('levmar', '_levmar' + ext)] + levmar_sources)]
-    if USE_CYTHON:
-        from Cython.Build import cythonize
-        ext_modules = cythonize(ext_modules)
-    ext_modules[0].include_dirs = ['levmar/levmar-2.6', np.get_include()]
-    if env['LAPACK']:
-        ext_modules[0].libraries += [env['LAPACK']]
-    if env['BLAS']:
-        ext_modules[0].libraries += [env['BLAS']]
+if is_win:
+    libs = ['cbia.lib.blas.dyn.rel.x64.12', 'cbia.lib.lapack.dyn.rel.x64.12'] if is_win else []
```

Comment:
those files have hardcoded version numbers in them. And as binaries, they have no place in a source distribution. And in a binary distribution (bdist wheel for example) I'm unsure whether it even would be allowed to redistribute?

Reply:
the file name has no version number. .12 refers to the MSVC compiler version.
It was a real pain to get these binaries dynamically linked. I can try letting conda-forge to the job, but I doubt it will work.

I got them from here (no license given): https://www.fi.muni.cz/~xsvobod2/misc/lapack/

Reply:
ok, anyhow, either we should statically link lapack for distribution as a binary wheel on PyPI or dynamically link using conda and have conda handle resolution of dependcies (I'm quite positive that this is the best way forward).